codebuild/spec/buildspec_sidetrail.yml

```diff
@@ -50,5 +50,3 @@ phases:
   post_build:
```

Comment:
Optionally for all of these with nothing else happening in `post_build`, we could just drop the whole phase.

Reply:
yes true. in the interest of not manually running the ci jobs again, i am inclined to address this in a follow up pr